sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -323,6 +324,10 @@ private static String getSdkVersionWithSnapshotInsteadOfBeta() {
         }
     }
 
+    public static class ChangeFeedWireFormatVersions {
```

Comment:
why do we need to capture what looks like an API version here?

Reply:
it's not needed but it is an improvement in the content that is returned - https://msdata.visualstudio.com/CosmosDB/_wiki/wikis/CosmosDB.wiki/34015/dotNET

Reply:
This should be an "enum" class then, if we ever plan to support other CF wire formats.